package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "knockout.autocomplete",
-  "version": "1.6.0",
+  "version": "1.7.0",
```

Comment:
This is maybe wrong, I don't know? .. @birkestroem talked about using `npm version` and then publishing the tags. If thats the case, I can just revert this commit - but let me know